BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift

```diff
@@ -173,30 +174,20 @@ public class BottomSheetViewController: UIViewController,
     }
 
     // MARK: - Private
-
-    @available(iOS 26.0, *)
-    private func setupDynamicBackground() {
-        let screenHeight = UIScreen.main.bounds.height
-        let seventyFivePercentScreenHeight = screenHeight * 0.75
-        let currentSheetHeight = childViewController.view.frame.height
-
-        if currentSheetHeight <= seventyFivePercentScreenHeight {
-            setupGlassEffect()
-        } else {
-            removeGlassEffect()
-            contentView.backgroundColor = themeManager.getCurrentTheme(for: windowUUID).colors.layer1
-        }
-    }
-
     @available(iOS 26.0, *)
     private func setupGlassEffect() {
         // Only add glass effect if it doesn't already exist
-        guard glassEffectView == nil else { return }
+        guard glassEffectView == nil else {
+            updateGlassEffectTint()
+            return
+        }
 
         let effectView = UIVisualEffectView()
 
         #if canImport(FoundationModels)
         let glassEffect = UIGlassEffect()
+        let theme = themeManager.getCurrentTheme(for: windowUUID)
+        glassEffect.tintColor = theme.colors.layer2.withAlphaComponent(UX.glassAlpha)
```

Comment:
I think it would be better to call `updateGlassEffectTint()` here so we only set the tint in one place.